skunkbooth/main.py

```diff
@@ -119,7 +131,21 @@ def CamDimensions(height: int, width: int) -> Tuple[int, int, int]:
                     Scene([PreviewFrame(screen, model=image_selection)], -1, name="Preview")
                 ]
 
-                screen.set_scenes(scenes)
+                screen.set_scenes(scenes, unhandled_input=global_shortcuts)
+
+            if screen.current_scene == scenes[2]:
+                event = screen.get_event()
+                if isinstance(event, KeyboardEvent):
+                    c = event.key_code
+                    layout = fFrame._layouts[1]
+                    if c == screen.KEY_HOME:
+                        fFrame.switch_focus(layout, 0, 0)
+                    elif c == screen.KEY_END:
+                        fFrame.switch_focus(layout, 0, len(fFrame.filterList)-1)
+                    elif c == screen.KEY_PAGE_UP:
```

Comment:
pgup and pgdown usually move by more than one arrow keypress, would be a full page but we don't have enough filters to page that far currently

Reply:
In commit 7efeeba, I created an empty function _skip_to_next_page with no implementation (pass) but which would contain this logic further down the line. Is this ok for now, or should I somehow emulate the pgup and pgdown navigation with the current set of filters?

Reply:
eh yeah sure this'll do for now, don't have enough filters to make this that relevant yet, we can make another easy ticket out of this